Bottom: be67ce9b8fa34c776e1508892b4c60e1aa62974d
Top:    b16e8fc78412c8b03d7b2d5ccdefd83f9428dca4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 04:35:21 +0100

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
